mono/mini/debugger-agent.c

```diff
@@ -3668,6 +3793,9 @@ process_event (EventKind event, gpointer arg, gint32 il_offset, MonoContext *ctx
 		case EVENT_KIND_METHOD_ENTRY:
 		case EVENT_KIND_METHOD_EXIT:
 			buffer_add_methodid (&buf, domain, (MonoMethod *)arg);
+#ifdef IL2CPP_MONO_DEBUGGER
+			buffer_add_long (&buf, il2cpp_seqpoint_id);
```

Comment:
Are we changing the socket protocol then?

Reply:
To be clear, I think this is bad as the client APIs ship with IDEs not runtime AFAIK.

Reply:
This was added to aid debugging from the client side. We discussed it a bit, but I don't think that we intend to change the API in anything we ship. @brianradunity we should probably do something with this before the PR lands. What do you think?

Reply:
This was to aid in debugging.  It allows us to see the sequence point id while debugging the tests and more quickly figure out where we are in the generated code.  I think we can wrap it in #define's to only compile this in if we are running the unit tests.  I was aware that it would break VSTU, etc.

Reply:
I've pushed a change to only add the sequence point ID to the buffer when we are running our tests.